README.md

```diff
@@ -236,3 +236,52 @@ In order to run the tests type the following to your shell:
 ```
 python -m unittest mona_sdk/tests/client_tests.py
 ```
+
+## Asynchronous Client
+
+Same services as the regular client are provided in a non-blocking version in the AsyncClient, in order to give you the option to embed the code you write using this package with asyncIO code.
+
+Everything described for the regular client applies also to the AsyncClient, with these additions:
```

Comment:
I think the phrasing can improve... If I understand correctly maybe - 
"When using AsyncClient, while all the regular (synchronous) client functions are still supported, you can simply add "_async" suffix to any function (e.g export_async() instead of export(); export_batch_async() instead of export_batch() etc). The async version of the methods accept the same parameter as the synchronous version, in addition to the following parameters:
...
"